ui/app/AppLayouts/Profile/views/EnsView.qml

```diff
@@ -318,25 +318,25 @@ Item {
         target: root.store.ens
         onTransactionWasSent: {
             //% "Transaction pending..."
-            toastMessage.title = qsTrId("ens-transaction-pending")
-            toastMessage.source = Style.svg("loading")
-            toastMessage.iconColor = Style.current.primary
-            toastMessage.iconRotates = true
-            toastMessage.link = `${root.store.etherscanLink}/${txResult}`
-            toastMessage.open()
+            Global.toastMessage.title = qsTrId("ens-transaction-pending")
+            Global.toastMessage.source = Style.svg("loading")
+            Global.toastMessage.iconColor = Style.current.primary
+            Global.toastMessage.iconRotates = true
+            Global.toastMessage.link = `${root.store.etherscanLink}/${txResult}`
+            Global.toastMessage.open()
```

Comment:
I might be wrong, but this looks like there can only be one `toastMessage` at a time.

I do agree that it makes sense for toastMessages to live in some global scope since they should be app section independent, but I also think that we should allow for multiple instances of this.

Does `toastMessage` have to be a prop of `Global`? And is it the one and only?

Reply:
Sounds good. Hopefully it's already filed in a github issue.

Reply:
When it's possible to open more than one toast messages, `Global.toastMessage` will become something like `Global.createToastMessage` or `Global.showToastMessage`, so in my opinion it's ok to keep `toastMessage` in `Global` for now. Let me know if you think differently!

Reply:
> Sounds good. Hopefully it's already filed in a github issue.

There's not.

> When it's possible to open more than one toast messages, Global.toastMessage will become something like Global.createToastMessage or Global.showToastMessage, so in my opinion it's ok to keep 

Yea, I tend to agree. I don't think it make sense to have various multiple places where `ToastMessage` is declared because they most likely should behave the same, no matter from where they have been called.

Just important that different parts of the app get access to the service to spin them up.

So let's leave as is for now.